packages/sdk/tests/base/auth.node.test.ts

```diff
@@ -86,7 +86,7 @@ describe('auth (node)', function () {
 			expect(sdk.storage.auth_expires).toBe(107000);
 			await tick(5000);
 
-			expect(scope.pendingMocks().length).toBe(1);
+			expect(scope.pendingMocks().length).toBe(0);
```

Comment:
Why 0? Shouldn't be 1, since `'/auth/refresh'` is not called yet?

Reply:
/auth/refresh is never called but it does expire at tick 5000. This tick is passed on line 87.